.github/workflows/prep-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Prepare Release PR
+
+# Manual trigger only
+on:
+  workflow_call:
+    inputs:
+      release_type:
+        description: Type of release. patch or minor (major if breaking)
```

Comment:
Is there any way for us to assert that this input is one of `{patch,minor,major}` to prevent invalid inputs (such as typos)?

Reply:
good call. we can now use `type: choice`

Reply:
actually it's already being enforced at the script level:

```
program.addOption(
  new Option('--release-type <releaseType>', 'release type')
    .choices(['patch', 'minor', 'major'])
    .makeOptionMandatory()
)
```
I'll keep it there as you can run the script from CLI as well